webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/editablemapping/EditableMappingService.scala

```diff
@@ -56,8 +56,8 @@ case class FallbackDataKey(
 )
 
 case class MinCutParameters(
-    segmentId1: Long,
-    segmentId2: Long,
+    partitionOne: List[Long],
+    partitionTwo: List[Long],
```

Comment:
Since we have segmentId1 and segmentId2 in so many places, I’d vote to change these variable names to partition1 and partition2 also, for the benefit of consistency.